rules-reviewed/technology-usage/embedded-framework-technology-usage.windup.xml

```diff
@@ -1036,5 +1036,19 @@
                 </technology-identified>
             </perform>
         </rule>
+        <rule id="technology-usage-embedded-framework-08900">
```

Comment:
This rule has been already committed in https://github.com/windup/windup-rulesets/pull/693